modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/skin/Utils.java

```diff
@@ -414,9 +425,17 @@ public static String computeClippedText(Font font, String text, double width,
         }
     }
 
-    public static String computeClippedWrappedText(Font font, String text, double width,
-                                            double height, double lineSpacing, OverrunStyle truncationStyle,
-                                            String ellipsisString, TextBoundsType boundsType) {
+    public static String computeClippedWrappedText(
+        Font font,
+        String text,
+        double width,
+        double height,
+        double lineSpacing,
+        OverrunStyle truncationStyle,
+        String ellipsisString,
+        AtomicBoolean textTruncated,
```

Comment:
Similarly, I recommend setting it to false as the first statement. In this case, I see that it will be set to false in the case where it makes it to the return statement at the end of the method without being truncated, but there is one early return where this isn't the case.